examples/ssh/windows_check_vpn_ext_hosts.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Domotz Custom Driver 
+ * Name: Windows check VPN connection
+ * Description: Check if a list of hosts are reachable from the endpoint where the script is run 
+ *   
+ * Communication protocol is SSH. Utilizing the native windows powershell command.
+ * 
+ * Tested on Windows Versions:
+ *  - Windows 10
+ *  - Microsoft Windows Server 2019
+ * 
+ * Powershell Version:
+ *  - 5.1.19041.2364
+ * 
+ * 
+ * Creates a Custom Driver Table with the following columns:
+ *  - Host
+ *  - Reachable
+ * 
+ *  Reachable can be [Yes, No]
+ * 
+**/
+
+// Define here the list of hosts/ips you would like to check
+var hostsToCheck = "www.notworkingnetaddress.com,www.google.com,www.microsoft.com.it,129.312.32.31" 
+
+// SSh command run on the endpoint
+var command="powershell -command \"Test-Connection -Cn "+hostsToCheck+" -BufferSize 16 -Count 1 -ea 0 -quiet \""
```

Comment:
If this is the only command executed, do you need it defined outside the sshConfig?